### PR TITLE
[UI] Description component

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description
This PR makes descriptions cool.

image

image

https://github.com/user-attachments/assets/31e4c23d-e3d0-405f-b3ac-31a26ced206a

| Before                                 | After
| -------------------------------------- | ---
|         image                               | image

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Segments 
3. Create new, set its name and a description of less than 70 characters
4. Save & Close
5. See the description is loaded open
6. Create another with more than 71 characters
7. See it's loaded closed by default

### CODE REVIEW

File: app/bundles/AssetBundle/Resources/views/Asset/details.html.twig [hunk 1]
```diff
@@ -43,13 +43,7 @@
...
         <div>
             <!-- asset detail header -->
-            <div class="pr-md pl-md pt-lg pb-lg">
-                <div class="box-layout">
-                    <div class="col-xs-10 va-m">
-                        <div class="text-white dark-sm mb-0">{{ activeAsset.getDescription()|purify }}</div>
-                    </div>
-                </div>
-            </div>
+            {% include '@MauticCore/Helper/description.html.twig' with {'description': activeAsset.description} %}
```
Comment (shinde-rahul) [inline app/bundles/AssetBundle/Resources/views/Asset/details.html.twig:62]: This is awesome: the reusable components. 🫶

**Review by shinde-rahul — APPROVED** [review]
The code changes look good to me.

File: app/bundles/AssetBundle/Resources/views/Asset/details.html.twig (hunk 1 above)
Comment (andersonjeccel) [reply app/bundles/AssetBundle/Resources/views/Asset/details.html.twig:62]: Yeah, I realized that making things reusable also allows us to ensure they're absurdly better in terms of UX as we won't have to customize many other hardcoded things, individually, later

my idea is to start creating components based on the type of information they aim to show (being unique for each)

**Review by abhisekmazumdar — APPROVED** [review]
LGTM
image

image

**Comment by escopecz** [comment]
@andersonjeccel there are some conflicts after merging some other PRs

**Comment by escopecz** [comment]
@andersonjeccel the e2e test is failing consistently in this case it seems. Can you check if this path wasn't changed?

https://github.com/mautic/mautic/blob/1f0e3e40dcd0e9b84b5c238de84d2996c22887ed/tests/_support/Page/Acceptance/EmailsPage.php#L15

**Comment by andersonjeccel** [comment]
@escopecz yep, fixed now

thanks for having a look :)

**Comment by andersonjeccel** [comment]
the error now seems to be the normal one

**Pull request merged**